inc/theme-hooks.php

```diff
@@ -125,3 +125,123 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+
+add_filter( 'alcatraz_primary_sidebar', 'alcatraz_sidebar_nav_menu', 5);
+/**
+ * Output the sidebar nav.
+ *
+ * @since  1.0.0
+ */
+
+function alcatraz_sidebar_nav_menu() {
+
+	global $post;
+
+	$args = array();
+
+	//defaults
+	$defaults = array(
+		'show_all'         => true,
+		'show_on_home'     => false,
+		'show_empty'       => false,
+		'sort_by'          => 'menu_order',
+		'a_heading'        => false,
+		'title'            => ''
+	);
+
+	$args = wp_parse_args( (array)$args, $defaults );
+
+	// Get clean param values.
+	$show_all         = $args['show_all'];
+	$show_on_home     = $args['show_on_home'];
+	$show_empty       = $args['show_empty'];
+	$sort_by          = $args['sort_by'];
+	$a_heading        = $args['a_heading'];
+	$title            = $args['title'];
+
+	if ( is_search() || is_404() ) {
+		return false; //doesn't apply to search or 404 page
+	}
+
+	if ( is_front_page() ) {
+		return false;	//if we're on the front page and we haven't chosen to show this anyways, leave
+	}
+
+	if(!$post->post_parent) {
+		$children = wp_list_pages("title_li=&child_of=".$post->ID."&echo=0");
```

Comment:
Let's make all calls to wp_list_pages consistent by passing an array of args rather than a string like this. See the call below on line 199 for an example.